pkg/values/value.go

```diff
@@ -45,6 +45,32 @@ func Wrap(v any) (Value, error) {
 	return nil, fmt.Errorf("could not wrap into value: %+v", v)
 }
 
+func MustWrap(v any) Value {
```

Comment:
Where would this be used?

Reply:
Initially, I thought that this would only be used in tests, but is it even possible for err to be not nil in map and list cases? If I understand correctly, we should always be able to match one of the types, so it should never error.

Reply:
Maybe but the callers only have the signature that includes an error to work with and shouldn't be assuming any implementation details (that could change in the future). If the panic case never executes, then there is no difference in practice, and if it does, we certainly want to know about it.